app/src/main/java/protect/budgetwatch/MainActivity.java

```diff
@@ -95,39 +87,36 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             startIntro();
             prefs.edit().putBoolean("firstrun", false).commit();
         }
+        PasswordManager pm = new PasswordManager(this);
+        if (!authorized && pm.isPasswordEnabled())
+            startAuthorization();
     }
 
-    static class MainMenuItem
-    {
+    static class MainMenuItem {
         public final int iconId;
         public final int menuTextId;
         public final int menuDescId;
 
-        public MainMenuItem(int iconId, int menuTextId, int menuDescId)
-        {
+        public MainMenuItem(int iconId, int menuTextId, int menuDescId) {
```

Comment:
Can you keep the style the same as before, namely where the { } are?

Reply:
Done